src/Randomizer.SMZ3.Tracking/Tracker.cs

```diff
@@ -787,8 +787,9 @@ public void ConnectToChat(string? userName, string? oauthToken, string? channel,
                 {
                     _chatClient.Connect(userName, oauthToken, channel ?? userName, id);
                 }
-                catch (Exception e)
+                catch (AggregateException e)
                 {
+                    _logger.LogError("Error in connection to Twitch chat", e);
```

Comment:
The parameters should be the other way around, now `e` is a format object that's not used in the string.

Reply:
... Whoops. I think I made this mistake elsewhere as well. I'll go through and fix ones I can find.